lib/node_modules/@stdlib/array/float16/lib/polyfill.js

```diff
@@ -0,0 +1,903 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var min = require( '@stdlib/math/base/special/min' );
+var max = require( '@stdlib/math/base/special/max' );
+var floor = require( '@stdlib/math/base/special/floor' );
+
+
+// MAIN //
+
+/**
+* Creates a new Float16Array.
+*
+* @constructor
+* @param {(number|Array|ArrayBuffer|TypedArray)} input - Input data or buffer
+* @param {NonNegativeInteger} [byteOffset=0] - Byte offset
+* @param {NonNegativeInteger} [length] - Number of elements
+* @throws {TypeError} Must provide valid input
+*/
+function Float16Array( input, byteOffset, length ) {
+    var i;
```

Comment:
This file should be all TAB indentation. Currently, you are using space indentation.